felix/calc/calc_graph_bench_test.go

```diff
@@ -0,0 +1,331 @@
+// Copyright (c) 2023 Tigera, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package calc
+
+import (
+	"encoding/binary"
+	"fmt"
+	"net"
+	"runtime"
+	"testing"
+	"time"
+
+	. "github.com/onsi/gomega"
+	v3 "github.com/projectcalico/api/pkg/apis/projectcalico/v3"
+	"github.com/sirupsen/logrus"
+
+	"github.com/projectcalico/calico/felix/config"
+	"github.com/projectcalico/calico/libcalico-go/lib/backend/api"
+	"github.com/projectcalico/calico/libcalico-go/lib/backend/k8s/conversion"
+	"github.com/projectcalico/calico/libcalico-go/lib/backend/model"
+	calinet "github.com/projectcalico/calico/libcalico-go/lib/net"
+)
+
+const numNamespaces = 100
+
+func BenchmarkInitialSnapshot200Local250kTotal10000TagPols(b *testing.B) {
+	benchInitialSnap(b, 250_000, 200, 10000, 0)
+}
+
+func BenchmarkInitialSnapshot200Local10kTotal1000NetsetPols(b *testing.B) {
+	benchInitialSnap(b, 10000, 200, 0, 1000)
+}
+
+func BenchmarkInitialSnapshot200Local10kTotal10000NetsetPols(b *testing.B) {
+	benchInitialSnap(b, 10000, 200, 0, 10000)
+}
+
+var (
+	keepAlive any
+	_         = keepAlive
```

Comment:
I understand what `keepAlive any` is for, but what does `_ = keepAlive` do?

Reply:
It's to keep the linter happy, otherwise it flags keepAlive as unused.